climada/util/api_client.py

```diff
@@ -578,12 +561,8 @@ def get_hazard(self, hazard_type, dump_dir=SYSTEM_DIR, max_datasets=1, **kwargs)
             Directory where the files should be downoladed. Default: SYSTEM_DIR
             If the directory is the SYSTEM_DIR, the eventual target directory is organized into
             dump_dir > hazard_type > dataset name > version
-        max_datasets : int, optional
-            Download limit for datasets. If a query matches is matched by more datasets than this
-            number, a ValueError is raised. Setting it to 0 or a negative number inactivates the
-            limit. Default is 10.
-        **kwargs :
-            additional parameters passed on to get_datasets
+       **kwargs :
```

Comment:
I am not convinced by the used of `kwargs` here. This is one of the most exposed methods from the API. Thus, the docstring should not be obscure. It should be very clear to the user what the input parameters are.